### PR TITLE
docs(Rebranding / Typography): Add documentation changes

### PR DESCRIPTION
**Pull request by saurabhdaware**
- Installation Step changes for new fonts
- Adding fonts to react-native storybook
- Renamed font everywhere from `Tasa Orbiter` --> `TASA Orbiter` since that is more accurate casing as per their defined font family.

### CODE REVIEW

File: packages/blade/docs/guides/Installation.stories.mdx [hunk 1]
```diff
@@ -40,125 +40,114 @@ Before you install the package, make sure that you have performed the following
...
+  }
+  `;
+  ```
 
 ### React Native Projects
 
-- Download `Lato` font from [here](https://fonts.google.com/share?selection.family=Lato:ital,wght@0,400;0,700;1,400;1,700)
-- Copy the font files and paste them under the directory `<project_root>/public/fonts`(**create the directory if it doesn't exist**)
-- Create a React Native config file at the root of your project - `<project_root>/react-native.config.js` and add the following content to it
+- Download fonts from <a href="https://github.com/razorpay/blade/blob/rebranding/typography-documentation/packages/blade/fonts/blade-fonts-react-native.zip">blade-fonts-react-native.zip</a> and unzip.
```
Comment (snitin315) [inline packages/blade/docs/guides/Installation.stories.mdx:114]: Change this link to point to the master branch when merging.

File: packages/blade/docs/guides/Installation.stories.mdx [hunk 2]
```diff
@@ -40,125 +40,114 @@ Before you install the package, make sure that you have performed the following
...
-  <?xml version="1.0" encoding="utf-8"?>
-    <font-family xmlns:app="http://schemas.android.com/apk/res-auto">
-    <font app:fontStyle="normal" app:fontWeight="400" app:font="@font/lato_regular"/>
-    <font app:fontStyle="normal" app:fontWeight="700" app:font="@font/lato_bold" />
-    <font app:fontStyle="italic" app:fontWeight="400" app:font="@font/lato_regular_italic" />
-    <font app:fontStyle="italic" app:fontWeight="700" app:font="@font/lato_bold_italic" />
-  </font-family>
-  ```
-- Navigate to `/android/app/src/main/java/com/<your_package_name>/MainApplication.java` and add the following contents to it
+  - Copy all font files from `ios-fonts` directory to `<project_root>/public/fonts` (**create the directory if it doesn't exist**)
```
Comment (snitin315) [inline packages/blade/docs/guides/Installation.stories.mdx:118]:
```suggestion
  - Copy all font files from the `ios-fonts` directory to `<project_root>/public/fonts` (**create the directory if it doesn't exist**)
```

File: packages/blade/.storybook/react-native/storybook.requires.js [hunk 3]
```diff
@@ -100,9 +100,23 @@ const getStories = () => {
...
     './src/components/Spinner/Spinner/Spinner.stories.tsx': require('../../src/components/Spinner/Spinner/Spinner.stories.tsx'),
-    './src/components/SpotlightPopoverTour/Tour.stories.tsx': require('../../src/components/SpotlightPopoverTour/Tour.stories.tsx'),
+    './src/components/SpotlightPopoverTour/docs/Tour.stories.tsx': require('../../src/components/SpotlightPopoverTour/docs/Tour.stories.tsx'),
+    './src/components/SpotlightPopoverTour/docs/TourDocs.stories.tsx': require('../../src/components/SpotlightPopoverTour/docs/TourDocs.stories.tsx'),
     './src/components/Switch/Switch.stories.tsx': require('../../src/components/Switch/Switch.stories.tsx'),
-    './src/components/Table/docs/Table.stories.tsx': require('../../src/components/Table/docs/Table.stories.tsx'),
+    './src/components/Table/docs/APIStories/TableAPI.stories.tsx': require('../../src/components/Table/docs/APIStories/TableAPI.stories.tsx'),
+    './src/components/Table/docs/APIStories/TableBodyAPI.stories.tsx': require('../../src/components/Table/docs/APIStories/TableBodyAPI.stories.tsx'),
```
Comment (snitin315) [inline packages/blade/.storybook/react-native/storybook.requires.js:107]: Maybe we should ignore Table stories as this component is not available for react-native users.

**Review by snitin315 — APPROVED** [review]
LGTM

File: packages/blade/docs/guides/Installation.stories.mdx (hunk 1 above)
Comment (saurabhdaware) [reply packages/blade/docs/guides/Installation.stories.mdx:114]: oh good catch. Yup I should use master only here

File: packages/blade/.storybook/react-native/storybook.requires.js (hunk 3 above)
Comment (saurabhdaware) [reply packages/blade/.storybook/react-native/storybook.requires.js:107]: These get added automatically when we run react native storybook. Might have to write some extra code to ignore them in native so we can let it be.

For now we do have a fallback handler in Table which just prints Table is not supported in react native if someone opens these stories

**Review by anuraghazra — APPROVED** [review]

**Pull request merged**